pkg/operator/staticpod/installerpod/cmd.go

```diff
@@ -224,6 +226,12 @@ func (o *InstallOptions) copySecretsAndConfigMaps(ctx context.Context, resourceD
 		return err
 	}
 
+	stagingDirBase := getStagingDir(resourceDir)
```

Comment:
in line 224 the installer will create the target dir. i think that for the certsync we delegated that to the atomicdir pkg. should we do the same here ?